java-frontend/src/main/java/org/sonar/java/se/constraint/ObjectConstraint.java

```diff
@@ -65,7 +65,7 @@ public boolean isNull() {
     return isNull;
   }
 
-  public boolean isInversible() {
+  public boolean isInversible(@Nullable Constraint otherConstraint) {
```

Comment:
![MAJOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-major.png) Remove this unused method parameter "otherConstraint". [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://dory.sonarsource.com/sonarqube/coding_rules#rule_key=squid%3AS1172)